README.md

```diff
@@ -184,40 +168,27 @@ This is executed as a single stage:
 
 `Drop Dynamic Table`
 
-# Dynamic Table Dimension
-
-The Coalesce Dynamic Table Dimension UDN is a versatile node that allows you to develop and deploy a single Dynamic Table Dimension or a DAG of Dynamic Tables in Snowflake. 
+<h2 id="dynamic-table-dimension">Dynamic Table Dimension</h2>
 
-[Dynamic tables](https://docs.snowflake.com/en/user-guide/dynamic-tables-about. ) are a new table type offered by Snowflake that allow data teams to use SQL statements to declaratively define the results of data pipelines. Dynamic tables simplify the process of creating and managing data pipelines by streamlining data transformations without having to manage Streams and Tasks. 
+The Coalesce Dynamic Table Dimension UDN is a versatile node that allows you to develop and deploy a single Dynamic Table Dimension or a DAG of Dynamic Tables in Snowflake.
 
-## Version Information
- * UDN Name: Dynamic Table Dimension
- * Package Name: 
+[Dynamic tables](https://docs.snowflake.com/en/user-guide/dynamic-tables-about) are a new table type offered by Snowflake that allow data teams to use SQL statements to declaratively define the results of data pipelines. Dynamic tables simplify the process of creating and managing data pipelines by streamlining data transformations without having to manage Streams and Tasks.
 
- * Certified Version: 1.1
- * Documentation Version: 1.1
- * Current Version: 1.1
- * Originally Developed by: John Gontarz
- * Current Owner: John Gontarz
- * Materialization Type: Dynamic Table/transient dynamic table
- * Create Template: Yes
- * Run Template: No
+### Dynamic Table Dimension Node Configuration
 
- * Deployment Strategy: Advanced
- * Deployment Method: ALTER when possible otherwise CREATE OR REPLACE
-
- * Test Enabled: No
-
-## Node Configuration
 The Dynamic Table Dimension has three configuration groups:
 
-* [Node Properties](#node-properties)
-* [Dynamic Table Options](#dynamic-table-options)
-* [Dimension Options](#dimension-options)
+* [Node Properties](#dynamic-table-dimension-node-properties)
+* [Dynamic Table Options](#dynamic-table-dimension-options)
+* [General Options](#general-dimension-options)
+
```

Comment:
This is missing "Dimension Options" config group

Reply:
@ghenkhaus Dimension options added and aligned them accordingly